skopt/optimizer/optimizer.py

```diff
@@ -467,7 +471,7 @@ def tell(self, x, y, fit=True):
                 # minimization starts from `n_restarts_optimizer` different
                 # points and the best minimum is used
                 elif self.acq_optimizer == "lbfgs":
-                    x0 = X[np.argsort(values)[:self.n_restarts_optimizer]]
+                    x0 = X[np.argsort(values)]
```

Comment:
Maybe someone with more randomness intuition can help out here: is this semantically the same? Picking (say) top five from a random list or sampling five points at random. Do the two cases have the same chances of finding a good point? I think by sampling many you have a higher chance of finding good ones ...

I *think* most papers actually talk about "pick five points at random and use those" and not "sample many and then use the top five".